packages/components/Tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,150 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { Platform, View } from 'react-native';
+import { Text } from '@fluentui-react-native/text';
+import { FocusZone } from '@fluentui-react-native/focus-zone';
+import { tabsName, TabsType, TabsProps, TabsState, TabsSlotProps, TabsRenderData, TabsContextData } from './Tabs.types';
+import { compose, IUseComposeStyling } from '@uifabricshared/foundation-compose';
+import { ISlots, withSlots } from '@uifabricshared/foundation-composable';
+import { settings } from './Tabs.settings';
+import { mergeSettings } from '@uifabricshared/foundation-settings';
+import { filterViewProps } from '@fluentui-react-native/adapters';
+import { foregroundColorTokens, textTokens, backgroundColorTokens } from '@fluentui-react-native/tokens';
+import { useSelectedKey } from '@fluentui-react-native/interactive-hooks';
+
+export const TabsContext = React.createContext<TabsContextData>({
+  selectedKey: null,
+  onTabsClick: (/* key: string */) => {
+    return;
+  },
+  getTabId: (/* key:string, index: number*/) => {
+    return null
+  },
+  updateSelectedTabsItemRef: (/* ref: React.RefObject<any>*/) => {
+    return;
+  },
+  tabsItemKeys: [],
+  views: null,
+});
+
+export const Tabs = compose<TabsType>({
+  displayName: tabsName,
+
+  usePrepareProps: (userProps: TabsProps, useStyling: IUseComposeStyling<TabsType>) => {
+    const {
+      label,
+      accessibilityLabel = userProps.label,
+      selectedKey,
+      headersOnly,
+      defaultSelectedKey,
+      getTabId,
+      componentRef = React.useRef(null),
+      isCircularNavigation,
+      ...rest
+    } = userProps;
+
+    // This hook updates the Selected TabsItem and calls the customer's onTabsClick function. This gets called after a TabsItem is pressed.
+    const data = useSelectedKey(selectedKey || defaultSelectedKey || null, userProps.onTabsClick);
+
+    const [selectedTabsItemRef, setSelectedTabsItemRef] = React.useState(React.useRef<View>(null));
+
+    const onSelectTabsItemRef = React.useCallback(
+      (ref: React.RefObject<View>) => {
+        setSelectedTabsItemRef(ref);
+      },
+      [setSelectedTabsItemRef],
+    );
+
+    const onChangeTabId = React.useCallback((key: string, index: number) => {
+      if (getTabId) {
+        return getTabId(key, index);
+      }
+      return `${key}-Tab${index}`;
+    }, []);
+
+    // Stores views to be displayed
+    const map = new Map<string, React.ReactNode[]>();
+
+    const state: TabsState = {
+      context: {
+        selectedKey: selectedKey ?? data.selectedKey,
+        onTabsClick: data.onKeySelect,
+        getTabId: onChangeTabId,
+        updateSelectedTabsItemRef: onSelectTabsItemRef,
+        views: map,
+      },
+      info: {
+        headersOnly: headersOnly ?? false,
+        label: !!label && Platform.OS === 'windows',
```

Comment:
Why is `Platform.OS === 'windows'` needed here?

Reply:
Because only the windows Tabs Control has a label on native examples found. Should we still include it for all platforms?

Reply:
But the Label prop is available to all other platforms (via Tabs.types.ts), correct? It wouldn't be user-friendly if we surface the prop to all platforms, and say a MacOS user sets the label prop, and then we don't use it. The user will get confused and most likely file a bug.

Since it's an optional prop (and pretty harmless one, at that), I say we use it for all platforms.

Reply:
Correct, added it back to all platforms.